xhr/formdata/get-from-domform.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html lang=en>
 <meta charset=utf-8>
-<title>FormData: get and getAll</title>
+<title>FormData: get and getAll (with dom form)</title>
```

Comment:
with form element*

This applies throughout and also to the file names. I stopped noting this particular comment, but there's more instances.